packages/components/src/components/Cell/Cell.tsx

```diff
@@ -48,20 +55,38 @@ export const Cell = React.forwardRef<View, React.PropsWithChildren<CellProps>>(
                     </View>}
                 </PressableHighlight>
             </View>
+        )
+        return (
+            (leftSwipeGroup || rightSwipeGroup) ?
+                <Swipeable
+                    overshootFriction={8}
+                    containerStyle={{ backgroundColor: styles.container.backgroundColor }}
+                    renderLeftActions={() => leftSwipeGroup && leftSwipeGroup.map((swipeItem, index) => (
+                        <CellSwipeItem key={`testL_${index}`} {...swipeItem} />
+                    ))}
+                    renderRightActions={() => rightSwipeGroup && rightSwipeGroup.map((swipeItem, index) => (
+                        <CellSwipeItem key={`test_${index}`} {...swipeItem} />
```

Comment:
should probably replace "test" with something else in the key names

Reply:
fixed in 06e1591